tests/jballerina-unit-test/src/test/resources/test-src/types/tuples/tuple_access_expr.bal

```diff
@@ -277,3 +277,237 @@ function testTupleAccessUsingUnionWithFiniteTypesNegative() {
     string|boolean f1 = tuple[index0];
     var f2 = tuple[index1];
 }
+
```

Comment:
Below will still crash
```ballerina
type Foo [string, int];

function name(Foo val, byte x) {
    _ = val[x];
}
```

Reply:
The PR only resolve the problem for integer datatypes up-to now. Will resolve this as well.